webapp/components/interop.js

```diff
@@ -319,32 +390,38 @@ class WPTInterop extends QueryBuilder(
     return labels;
   }
 
-  computeTestRuns(metadata) {
-    return metadata && metadata.test_runs;
-  }
-
-  computeTestPaths(fileMetrics) {
-    const paths = fileMetrics && fileMetrics.map(m => m.dir) || [];
+  computeTestPaths(searchResults) {
+    const paths = searchResults && searchResults.results.map(r => r.test) || [];
     return new Set(paths);
   }
 
-  computeFileMetrics(allMetrics) {
-    let fileMetrics = [];
-    for (const metric of allMetrics.data) {
-      if (this.computePathIsATestFile(metric.dir)) {
-        fileMetrics.push(metric);
+  precomputedInteropLoaded(precomputedInterop) {
+    const searchResults = {
+      runs: this.testRuns,
+      results: [],
+    };
+    if (precomputedInterop) {
```

Comment:
Can this be false, and if so is an else-branch not needed?

Reply:
Tweaked.